source/Playnite/Common/Programs.cs

```diff
@@ -56,13 +56,42 @@ public partial class Programs
             @"unins\d+"
         };
 
+        private static readonly string[] configAndRedistsExecutablesMasks = new string[]

```

Comment:
I don't see a good reason to have two separate lists. You can have both set up to be regex matches.

Reply:
I thought it would make it easier to maintain, in case more executables are added in the future and that it would also make it clearer to understand what each executable is. It's also the reason why I didn't put the new masks in the existing list for uninstallers. I also think that using a regex for the specific engine executables is not needed since we can know the exact name of the executables. But let me know if you still think that it should use a new single list to change the PR